src/helpers/text_helper.ts

```diff
@@ -224,6 +228,90 @@ export function getContextFontSize(font: string): Pixel {
   return Number(font.match(pxRegex)?.[1]);
 }
 
+export function fittingString(ctx, str: string, maxWidth: number) {
+  var width = ctx.measureText(str).width;
```

Comment:
doesn't the size also depends on the font weight (bold)?

Reply:
Yes, but the canvas should have been correctly configurated to take that into account before